pkgs/by-name/qr/qrencode/package.nix

```diff
@@ -20,21 +22,32 @@ stdenv.mkDerivation (finalAttrs: rec {
   ];
 
   src = fetchurl {
-    url = "https://fukuchi.org/works/qrencode/qrencode-${version}.tar.gz";
-    sha256 = "sha256-2kSO1PUqumvLDNSMrA3VG4aSvMxM0SdDFAL8pvgXHo4=";
+    url = "https://github.com/fukuchi/libqrencode/archive/refs/tags/v${version}.tar.gz";
+    hash = "sha256-U4W8G4wvIPO5HSWL+MzIz2ICOTXfLSZ2tbZwSfMaBJw";
   };
 
   nativeBuildInputs = [ pkg-config ];
 
   buildInputs = [
+    autoconf
+    automake
     libiconv
     libpng
+    libtool
   ];
 
-  nativeCheckInputs = [ SDL2 ];
-
   doCheck = false;
 
+  preConfigure = ''
+    test -d use || mkdir use
+    test -d m4 || mkdir m4
+    ${autoconf}/bin/autoheader
+    ${automake}/bin/aclocal
+    ${libtool}/bin/libtoolize --automake --copy
+    ${automake}/bin/automake --add-missing --copy
+    ${autoconf}/bin/autoconf
+  '';
+
```

Comment:
Have you tried autoreconfHook?

Reply:
I didn't know about that! Thanks for the tip! `autoreconfHook` works just fine. I'll update the PR shortly.